.github/workflows/build-openapi.yaml

```diff
@@ -0,0 +1,37 @@
+name: Run .NET Project with Swagger
```

Comment:
Why do we need a separate workflow for generating an openapi spec?

You could simply add the "--swagger" flag to the regular build workflow, so it is always included.

The way you do it now is building the same app twice. Once for deploying it, and another time to generate an openapi spec.